src/components/Menu/CompositeMenu/index.js

```diff
@@ -29,49 +29,60 @@ export default class Menu extends React.Component {
     es6ClassBindAll(this);
 
     this.state = {
-      filteredItems: undefined,
+      isOpen: false,
     };
   }
 
-  onFilterItems(filteredItems) {
-    this.setState({ filteredItems });
+  componentDidMount() {
+    mediaQueryList.addListener(this.unlockRootScroll);
   }
 
-  renderMenu() {
-    const { items } = this.props;
-    const { filteredItems } = this.state;
-
-    if (!filteredItems) {
-      return (
-        <TreeMenu items={items} />
-      );
-    }
+  componentWillUnmount() {
+    mediaQueryList.removeListener(this.unlockRootScroll);
+  }
 
-    if (_.isEmpty(filteredItems)) {
-      return (
-        <Alert color="secondary">검색 결과가 없습니다.</Alert>
-      );
-    }
+  unlockRootScroll(e) {
+    const { isOpen } = this.state;
+    lockRootScroll(isOpen && !e.matches);
+  }
 
-    return (
-      <TreeMenu items={filteredItems} forceExpand />
-    );
+  toggle() {
+    const isOpen = !this.state.isOpen;
+    this.setState({
+      isOpen,
+    });
+    lockRootScroll(isOpen);
```

Comment:
On small size screen, This locks body's scroll when menu is open. But this may cause a problem when users do not apply appropriate CSS as suggested in `example/src/styles.css`. Do we need to force users to apply specific CSS or provide some options to turn off this?